source/guides/clear/mixer.rst

```diff
@@ -256,11 +256,18 @@ these include the :command:`native-kernel` bundle that is intended to be
 used on a bare metal system instead of a VM. So we will modify the default
 bundle set to get a smaller kernel image, which will also be faster to load.
 
-The only bundles available to :command:`swupd` for a given release are those
-that were added to the mix during build time. A mix doesn’t automatically
-inherit upstream bundles.
+In this example, we introduce two type of bundles: ``package bundles`` that
+contain the list of RPMs; and ``content bundles`` that contain the path to 
+the local content. We recommend that the local content resides outside the 
+mixer work space.   
```

Comment:
Add a line: 

Mixer now supports creation of third-party bundles out of any content you want without the need to package it into an RPM format first.  Refer to `swupd`_ [add link]